### PR TITLE
Fix typos and plurals

### PR DESCRIPTION
**Pull request by francescofact**
fixed some English on the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,16 +86,16 @@ before submit the code check the Java format with the following command on the r
 ./gradlew verifyGoogleJavaFormat
 ```
 
-It any error are reported please run the following command to try to fix it
+If any error are reported please run the following command to try to fix them
```
Comment (vincenzopalazzo) [inline README.md:89]:
```suggestion
If any error is reported please run the following command to fix them
```

No try here if the syntax it is correct the linter make the job alone :-) (lukely)

File: README.md [hunk 2]
```diff
@@ -86,16 +86,16 @@ before submit the code check the Java format with the following command on the r
...
 
 ```bash
 ./gradlew googleJavaFormat
 ```
 
-p.s: The gradle plugin work with all the JDK version >= 9 (or better with java byte code version compatible with the version  55.0)
+p.s: The gradle plugin works with all the JDK versions >= 9 (or better with java byte code version compatible with the version  55.0)
```
Comment (vincenzopalazzo) [inline README.md:95]:
```suggestion
P.S: The gradle plugin works with all the JDK versions >= 9 (or better with java byte code version compatible with the version  55.0)
```

**Review by vincenzopalazzo — CHANGES_REQUESTED** [review]
LGTM @francescofact,

Only a small change request. But all looks good

**Comment by francescofact** [comment]
ok, changes applied!

**Review by vincenzopalazzo — APPROVED** [review]
LGTM, thanks for your amazing work.

@vincenzopalazzo review this PR.

**Pull request merged**